src/components/Questions/Questions.tsx

```diff
@@ -1,66 +1,26 @@
 import React from 'react';
 import QuestionCard from '../QuestionCard/QuestionCard'
+import { QuestionsType as QuestionDataType} from '../App/App'
```

Comment:
Great to see that you're able to reuse the question type throughout your app. It's really cleaning things up.